activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,32 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Order can be used to sort the results of a query.
+    #
+    # #order accepts arguments in one of several formats.
+    #
+    # === symbols
+    #
+    # The symbol represents the name of the column you want to order the results by.
     #
     #   User.order(:name)
     #   # SELECT "users".* FROM "users" ORDER BY "users"."name" ASC
     #
+    # By default the column used is ordered in ascending order.
+    # If you want descending order, then you can pass +:desc+ to the column name.
+    #
     #   User.order(email: :desc)
     #   # SELECT "users".* FROM "users" ORDER BY "users"."email" DESC
     #
+    # Multiple columns can be passed this way, and they will be sorted in the order specified.
+    #
     #   User.order(:name, email: :desc)
     #   # SELECT "users".* FROM "users" ORDER BY "users"."name" ASC, "users"."email" DESC
     #
+    # === strings
+    #
+    # Strings are passed directly to the database.
```

Comment:
I like how we're dividing the argument types into different sections now.  I think starting each section with an answer to "When would I use that instead?" would be good.  For this section, something like:

```suggestion
    # Strings are passed directly to the database, allowing you to specify 
    # simple SQL expressions.
```